src/main/java/com/epam/javacourses2016/task9/SolverTask9.java

```diff
@@ -16,7 +20,29 @@
      * @param input Анализируемый файл.
      * @return Множество полученных слов.
      */
+
     public HashSet<String> getUniqueWords(File input) {
-        return null;
+        List<String> lines = null;
+        try {
+            lines = Files.readAllLines(input.toPath(), StandardCharsets.UTF_8);
```

Comment:
:snail: 
We dont need that buffer actually, single Set with custom comparator should be enough